calico_test/tests/st/test_base.py

```diff
@@ -38,9 +41,17 @@ def setUp(self):
 
         # Delete /calico if it exists. This ensures each test has an empty data
         # store at start of day.
-        subprocess.check_output(
-            "curl -sL http://%s:2379/v2/keys/calico?recursive=true -XDELETE"
-            % self.ip, shell=True)
+        etcd_ssl_type = os.getenv("ETCD_SECURITY_TYPE", None)
+        if etcd_ssl_type == "ssl":
```

Comment:
Ah, yes.  Now that the HOST_CERT variable is passed in I can remove the ETCD_SECURITY_TYPE variable.